config/olm/openshift/current/manifests/dynatrace-oneagent-operator_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Delete this whole current directory. It doesn't seem to be used at all.

Reply:
Deleted the directory and the corresponding bundle file